README.md

```diff
@@ -80,92 +90,82 @@ Alternatively, you can install it directly from git:
 cargo install --git https://github.com/nimiq/core-rs-albatross.git
 ```
 
-## Documentation
-Extensive documentation explaining how the protocol is built up, the JSON-RPC specification, how to get started building applications on top of Nimiq and more can be found at the [Nimiq Developer Center](https://www.nimiq.com/developers/).
-
-## Software Integrity and Authenticity
-You can refer to the [documentation](./build/README.md) to learn more about reproducing Nimiq software for yourself as well as checking software signatures in order to verify the integrity and authenticity of software.
-
-## Usage
-
-After installation, you can run the client directly, like this:
+### Configuration
+You need a configuration file to customize your node according to your specific requirements. Follow one of the methods below to create and edit your configuration file.
 
+**Option A**
+The configuration file is generated automatically and in a specific location.
+1. Generate the configuration file with the following command:
 ```bash
 cargo run --release --bin nimiq-client
 ```
-
-### Configuration
-
-By default the client will look for a configuration file in `$HOME/.nimiq/client.toml`.
-In order to create this file yourself, you can use the example config file as follow:
-
+This generates a sample file and places it in a folder `./nimiq`.
+2. Copy the sample configuration file into a new file in the same directory where you will edit it according to your needs:
 ```bash
-cargo run --release --bin nimiq-client                         # Run the client. This will create the example config file.
-cp $HOME/.nimiq/client.example.toml $HOME/.nimiq/client.toml   # Create your config from the example.
-nano $HOME/.nimiq/client.toml                                  # Edit the config. Explanations are included in the file.
+cp $HOME/.nimiq/client.example.toml $HOME/.nimiq/client.toml 
```

Comment:
if the sample configuration file was generated as above, then this is not necessary